parser.cc

```diff
@@ -1466,7 +1466,9 @@ Input::Index StringInput::size() const
 	return str.size();
 }
 
-#ifdef __unix__
+const std::size_t Input::static_buffer_size;
```

Comment:
This line seems important on Mac?  You didn't mention it in the commit comments.

Reply:
I think the key thing here is that it's being moved outside of the `#if` block.  Without that, it's breaking on any platform that doesn't hit the UNIX code paths.

Reply:
So getting the `static_buffer_size` is actually what I was tring to accomplish, which would be solved by either moving it out of the ifdef or fixing the condition.

I've made this clearer in the PR, and also actually fixed the ifdef correctly.